src/cascadia/TerminalCore/TerminalDispatch.cpp

```diff
@@ -151,6 +240,12 @@ bool TerminalDispatch::InsertCharacter(const size_t count) noexcept
 // Return Value:
 // True if handled successfully. False otherwise
 bool TerminalDispatch::EraseInDisplay(const DispatchTypes::EraseType eraseType) noexcept
+try
 {
     return _terminalApi.EraseInDisplay(eraseType);
 }
+catch (...)
+{
+    LOG_CAUGHT_EXCEPTION();
+    return false;
+}

```

Comment:
Are all these try/catch blocks just a temporary thing? I was rather hoping the long term plan was to use exceptions instead of return values for error reporting (at least outside the legacy APIs). Just being able to rid of the return value checking all over the place would make things so much more readable. And with exceptions you can also more easily differentiate between different failure cases (e.g. an unsupported operation vs an implementation failure).

Reply:
Sure, I'd be happy to have it move entirely over to an exception based model. This was just the style at the time. I don't mean to put more work on you, but you seem to relish in it: if you would like to file a task and migrate all of the VT-related things from return-code-based to exception-based, I'd be happy to call it "code health".

Reply:
Serves me right for opening my big mouth. :) I didn't really have a specific plan for this, but I'll do some investigating on the weekend and file an issue when I have a better idea of how I think it might be achieved.